database/migrations/1659423228079_users.ts

```diff
@@ -0,0 +1,27 @@
+import BaseSchema from '@ioc:Adonis/Lucid/Schema'
+
+export default class UsersSchema extends BaseSchema {
+  protected tableName = 'users'
+
+  public async up() {
+    this.schema.createTable(this.tableName, (table) => {
+      table.increments('id').primary()
+      table.string('email', 255).notNullable().unique()
+      table.string('password', 180).notNullable()
+      table.string('remember_me_token').nullable()
+
+      /**
+       * Uses timestampz for PostgreSQL and DATETIME2 for MSSQL
+       */
+      table.timestamp('created_at', { useTz: true }).notNullable()
+      table.timestamp('updated_at', { useTz: true }).notNullable()
+
+      // Indexing
+      table.index(['id', 'email'])
```

Comment:
`id` already has `primary` key. `email` already has `unique` key.  I think we don't need the above index.